code/modules/client/verbs/ooc.dm

```diff
@@ -114,14 +114,15 @@ GLOBAL_VAR_INIT(mentor_ooc_colour, YOGS_MENTOR_OOC_COLOUR) // yogs - mentor ooc
 			oocmsg += "OOC:</span> <EM>[keyname]:</EM> <span class='message linkify'>[msg]</span></span></font>" // Footer for an admin or AO's OOC.
 			oocmsg_toadmins = oocmsg
 	else
-		oocmsg = "<span class='ooc'>[is_donator(src) ? "(Donator)" : ""]"
 		if(is_mentor()) // If the speaker is a mentor
+			oocmsg = "<span class='ooc'>["(Mentor)"]"
```

Comment:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

```suggestion
			oocmsg = "<span class='ooc'>(Mentor)"
```

Reply:
Now that is big brain

Reply:
what the fuck

Reply:
How the fuck does this compile